simcon_project/users/models/student.py

```diff
@@ -1,13 +1,9 @@
 from django.db import models
-from conversation_templates.models.conv_template import ConversationTemplate
-from conversation_templates.models.template_response import TemplateResponse
+from .custom_user import CustomUser
 
 
 class Student(CustomUser):
-    template_response = models.ForeignKey(TemplateResponse, on_delete=models.CASCADE)
```

Comment:
I assume this was similar reasoning that was applied to Researcher, but why are we getting rid of all of these fields?

Reply:
we had the relationship backwards for a lot of the fields. ForeignKey is used to create a Many to One relationship and there isnt a One to Many option so that forces us to create the relationship in the model that is the "Many" side. Specifically, we moved the line that creates the relationship between template responses and a student to the template_responses model. We had to do it that way because there are Many responses to one student so the relationship has to be created in that model, not the Student model.